lib/Sema/CSSimplify.cpp

```diff
@@ -10993,6 +10999,11 @@ ConstraintSystem::SolutionKind ConstraintSystem::simplifyFixConstraint(
 
     return SolutionKind::Solved;
   }
+          
+  case FixKind::AllowWrappedValueMismatch: {
+    if (recordFix(fix)) return SolutionKind::Error;
+      return SolutionKind::Solved;
```

Comment:
Nit: formatting is incorrect here, I'd suggest to use a ternary - `return recordFix(fix) ? SolutionKind::Error : SolutionKind::Solved;`